### PR TITLE
Add Static Progress Page

### PR DESCRIPTION
**Pull request by ike-elue**
Add:
- progress.html; static version of page (part of dynamic version coming in a future pull request)
- progress_disp.js; handles logic that creates progress bar and goal step display
- example pictures of possible progress page (in progress page images)

Wide Screen
image
image

Normal Screen
image
image

Update:
- index.html and style.css; to add progress page

### CODE REVIEW

**Review by GabrielGonzalezDiaz — APPROVED** [review]

**Review by lupengscu — APPROVED** [review]

**Review by piercedwilson — APPROVED** [review]

File: capstone/fitness/src/main/webapp/progress.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+            <li class="page-item"><a class="page-link" href="#">Next</a></li>
+          </ul>
+        </div>
+      </div><br>
+      <div class="row">
+        <!-- Goal Steps -->
+        <!--TODO(ijelue): Add Servlet and javascript for automatic generation of goal step buttons. -->
+        <div id="model">
+          <button type="button" class="btn btn-light" onclick="displayGoalStep()">Start</button><br>
+          <div id="button0" class="goal-desc"></div>
```
Comment (sliang3) [inline capstone/fitness/src/main/webapp/progress.html]: Consider naming this something more descriptive.

**Review by sliang3 — APPROVED** [review]
The black text is a little hard to read right now so I would suggest making the background a lighter grey maybe? Feel free to fix it later though.

File: capstone/fitness/src/main/webapp/progress.html (hunk 1 above)
Comment (ike-elue) [reply capstone/fitness/src/main/webapp/progress.html]: done

**Comment by ike-elue** [comment]
> The black text is a little hard to read right now so I would suggest making the background a lighter grey maybe? Feel free to fix it later though.

Will probably change the text to white, but this will be decided towards presentation time.

**Pull request merged**